src/main/java/uk/gov/ons/census/casesvc/service/CaseService.java

```diff
@@ -142,6 +146,7 @@ private CollectionCase createCollectionCase(Case caze, Address address) {
     collectionCase.setFieldCoordinatorId(caze.getFieldCoordinatorId());
     collectionCase.setFieldOfficerId(caze.getFieldOfficerId());
     collectionCase.setCeExpectedCapacity(caze.getCeExpectedCapacity());
+    collectionCase.setAddressInvalid(caze.isAddressInvalid());
```

Comment:
move below collectionCase.setRefusalReceived(caze.isRefusalReceived());

Reply:
why?

Reply:
receiptReceived and refusalReceived are not in the event dictionary, so frankly it's _those_ lines which are clearly in the wrong place, because they should appear after the comment which says "Below this line is extra data potentially needed by Action Scheduler"